quickcomm/models.py

```diff
@@ -86,14 +97,26 @@ def save(self, *args, **kwargs):
             Inbox.objects.create(content_object=self, author=self.following, inbox_type=Inbox.InboxType.FOLLOW)
 
         return saved
+    def delete(self, *args, **kwargs):
+        if Inbox.objects.filter(content_type=ContentType.objects.get_for_model(self), object_id=self.id).exists():
+            print("Yes")
+            Inbox.objects.filter(content_type=ContentType.objects.get_for_model(self), object_id=self.id).delete()
+        deleted=super(Follow,self).delete(*args,**kwargs)
+        return deleted
 
 
+    
+
     def is_bidirectional(self):
         """Returns true if the follow is bidirectional."""
         return self.following.is_following(self.follower)
 
     def __str__(self):
         return f"{self.follower.__str__()} follows {self.following.__str__()}"
+class follow_request(models.Model):
```

Comment:
Should probably stick the current class naming conventions, such that this would be `FollowRequest`